planner/core/common_plans.go

```diff
@@ -191,7 +191,7 @@ type Execute struct {
 
 	Name          string
 	UsingVars     []expression.Expression
-	PrepareParams []types.Datum
+	PrepareParams []types.Datum // parsed parameter values, only used for binary protocol
```

Comment:
What about commenting on these two lines together?
The `UsingVars` is for the txt protocol and the `PreparedParams` is for the binary protocol.

Reply:
Updated, PTAL.